resources/firewall.go

```diff
@@ -0,0 +1,27 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type Firewall struct{}
```

Comment:
I don't see any readme or example.tf changes - do we want to add for firewall etc?